assets/sass/components/tour-tooltip/_googlesitekit-tour-tooltip.scss

```
@@ -220,3 +220,65 @@
 	margin-top: 42px !important;
 	top: 0 !important;
 }
+
+// Setup Flow Refresh variant styles.
+.googlesitekit-tour-tooltip--setupFlowRefresh {
+
+	.googlesitekit-tooltip-title {
+		font-family: $f-primary;
+		font-size: $fs-title-lg;
+		font-weight: $fw-medium;
+		letter-spacing: $ls-xxs;
+		line-height: $lh-title-lg;
+		margin-bottom: 8px;
+	}
+
+	.googlesitekit-tooltip-content {
+		font-family: $f-primary;
+		font-size: $fs-body-md;
+		font-weight: $fw-normal;
+		letter-spacing: $ls-s;
+		line-height: $lh-body-md;
+	}
```

1. According to the Figma designs, typography for this should be `body-sm`.
2. I don't think it is necessary to specify the font family and weight.
3. Letter spacing should be `$ls-xs`.


```suggestion
	.googlesitekit-tooltip-content {
		font-size: $fs-body-sm;
		letter-spacing: $ls-xs;
		line-height: $lh-body-sm;
	}
```